uvtools/utils.py

```diff
@@ -115,8 +115,8 @@ def FFT(data, axis, taper=None, **kwargs):
        An array of data, assumed to not be ordered in the numpy FFT convention.
        Typically the data_array of a UVData object.
 
-    axis : int
-        The axis to perform the FFT over.
+    axis : int, optional
```

Comment:
Assuming this means for now we don't support delay and fringe rate at the same time?

Reply:
That's right; to do both you would need to do two successive calls of `FFT`. If you'd like it changed, then I can change it, but it'll have a few downstream consequences I'll have to patch up.

Reply:
I figure I should note that the intention here is that `FFT` is a convenience function for performing a FFT, with optional tapering, that handles the shifting under the hood. I was mainly fed up with having to do `np.fft.fftshift(np.fft.fft(np.fft.fftshift(...)))` whenever I wanted to Fourier transform something, and wanted to have the option to apply a taper without doing the extra work of generating the window and so on.

Reply:
Are things flexible enough to add 2D FFT down the road without changing interfaces much? If so, then its fine to file an issue and leave it for the future.

Reply:
It's possible. My preferred way of doing it would be to make a thin wrapper `FFT2D`, rather than make the code in `FFT` messy, but I suppose that details can be discussed later

Reply:
Ok. Please open an issue to put down a marker.